### PR TITLE
bug: tabulate viz & testing/usage examples

### PR DESCRIPTION
**Pull request by lizgehret**
This PR fixes a couple of small bugs, and adds some additional testing.

**The following bugs have been fixed:**

- A bug in the `tabulate` visualizer that would fail to create a visualization if 'id' was not present within the input table. This machinery worked with the ancombc R package version 3.15, but the output was modified slightly in 3.16 and instead of an 'id' column, there is now a 'taxon' column that represents the feature IDs. The index column has been re-named 'feature-id' and the index is pulled by column location instead of 'id' within the visualizer.

- A bug that unintentionally writes the `input.biom.tsv` and `input.map.txt` files (used for ancombc inputs) to the user's CWD. This was originally added for de-bugging purposes and was never removed.

**The following tests have been added:**

- A usage example for `qiime composition tabulate`

- A unit test for the FrictionlessCSV -> DataFrame transformer

**Some additional misc. items within this PR include:**

- Updating naming conventions for usage examples, tests, and test data to specify the ancombc method (to prevent confusion between ancom and ancombc).

- Updating the copyright year to 2023

### CODE REVIEW

**Review by Oddant1 — APPROVED** [review]
I'm not entirely sure what the issue was, but this looks good to me.

**Comment by lizgehret** [comment]
Okay I think this is ready for a final review @ebolyen - I'm going to open a separate PR for the usage example that includes back ticks in the formula once I have #100 sorted, but the rest of this should be all set.

File: q2_composition/assets/run_ancombc.R [hunk 1]
```diff
@@ -124,12 +124,19 @@ fit <- ancombc(data = data, formula = formula, p_adj_method = p_adj_method,
...
-lfc   <- cbind(feature_id = rownames(fit$res$lfc), fit$res$lfc)
-se    <- cbind(feature_id = rownames(fit$res$se), fit$res$se)
-w     <- cbind(feature_id = rownames(fit$res$W), fit$res$W)
-p_val <- cbind(feature_id = rownames(fit$res$p_val), fit$res$p_val)
-q_val <- cbind(feature_id = rownames(fit$res$q_val), fit$res$q_val)
+lfc   <- cbind(fit$res$lfc)
+se    <- cbind(fit$res$se)
+w     <- cbind(fit$res$W)
+p_val <- cbind(fit$res$p_val)
+q_val <- cbind(fit$res$q_val)
```
Comment (ebolyen) [inline q2_composition/assets/run_ancombc.R:139]: This cbind doesn't do anything anymore (I hope!).

Comment (lizgehret) [reply q2_composition/assets/run_ancombc.R:139]: Had to double check that (because R...), but you're right! Thanks @ebolyen!

**Comment by lizgehret** [comment]
**Note**: the dataloaf output's index has been changed from `feature-id` to `id` since we are discouraging the use of dashes in metadata columns when using ancombc (due to the issues mentioned above).

**Pull request merged**